nerdlets/top-nerdlet/process-details.js

```diff
@@ -77,19 +98,48 @@ export default class ProcessDetails extends React.PureComponent {
   renderSummaryPanel() {
     const { entity } = this.props;
     const select = ['commandLine', 'commandName', 'parentProcessId']
-      .map(s => `latest(${s})`)
+      .map(s => `latest(${s}) as ${s}`)
       .join(', ');
-    const nrql = this.metricQuery(select, '');
+    const nrql = this.summaryPanelQuery(select);
+
     return (
-      <NrqlQuery accountId={entity.accountId} query={nrql} formatType="raw">
-        {({ loading, data }) => {
-          if (loading) return null;
-          const { results } = data;
-          if (!results || results.length == 0) return null;
+      <NerdGraphQuery query={nrql}>
+        {({ loading, error, data }) => {
+          if (loading) {
+            return <Spinner style={{ height: '110px' }} />;
```

Comment:
What about just fillContainer vs. a custom style?

Reply:
We can do that. The difference is fillContainer covers the entire right panel (summary + charts) whereas this height restriction made it so the spinner covered just the summary component (i.e. less overall "flashing" when a new process is selected). I'm good to change it to fillContainer if we want to favor using the built-in props.